src/PKSim.Core/Snapshots/Mappers/ExtendedPropertyMapper.cs

```diff
@@ -21,10 +21,11 @@ public override Task<SnapshotExtendedProperty> MapToSnapshot(ModelExtendedProper
             snapshot.Name = extendedProperty.Name;
             snapshot.FullName = SnapshotValueFor(extendedProperty.FullName);
             snapshot.ReadOnly = SnapshotValueFor(extendedProperty.ReadOnly);
+            snapshot.Type = mapExtendedPropertyType(extendedProperty.Type);
```

Comment:
where is actually the type of extended properties set initially?
E.g. wehn importing observed data: is the type of all the metadata set anywhere?

Reply:
Obs data is always string

Reply:
ok